src/Board.hpp

```diff
@@ -1,7 +1,9 @@
 #pragma once
+#include "FieldsEnum.hpp"
```

Comment:
Not used? To be removed.

Reply:
used in this line:
> using BLine = std::vector<FieldsEnum>;

Reply:
Ok

Reply:
Yes, it was - has been removed in this commit - 6bdd875